### PR TITLE
creeperheal changes

### PR DESCRIPTION
**Pull request by erik1988**
Since creeperheal has some errors on 1.9 and 1.10 i have made some changes.

### CODE REVIEW

File: src/main/java/com/nitnelave/CreeperHeal/block/ExplodedBlockManager.java [hunk 1]
```diff
@@ -176,7 +176,7 @@ public static void processExplosion(List<Block> originalBlockList, Location loca
         Bukkit.getPluginManager().callEvent(event);
         if (event.isCancelled())
             return;
-        List<Block> processList = new ArrayList(event.getBlocks());
+        List<Block> processList = new ArrayList<Block>(event.getBlocks());
```
Comment (nitnelave) [inline src/main/java/com/nitnelave/CreeperHeal/block/ExplodedBlockManager.java]: Why is this needed?

File: src/main/java/com/nitnelave/CreeperHeal/command/CreeperCommandManager.java [hunk 2]
```diff
@@ -305,7 +305,7 @@ private void integerCmd(CfgVal key, String[] args, String help, String unit,
      *            to all worlds.
      */
     private void
-        forceCmd(String[] args, @SuppressWarnings("unused") String msg, CommandSender sender,
+        forceCmd(String[] args, String msg, CommandSender sender,
```
Comment (nitnelave) [inline src/main/java/com/nitnelave/CreeperHeal/command/CreeperCommandManager.java]: Again, why remove the annotation?

File: src/main/java/com/nitnelave/CreeperHeal/block/ExplodedBlockManager.java (hunk 1 above)
Comment (erik1988) [inline src/main/java/com/nitnelave/CreeperHeal/block/ExplodedBlockManager.java]: Its been awhile, but i think eclipse suggested the edit. 
You probably know this better than me tho...

File: src/main/java/com/nitnelave/CreeperHeal/command/CreeperCommandManager.java (hunk 2 above)
Comment (erik1988) [inline src/main/java/com/nitnelave/CreeperHeal/command/CreeperCommandManager.java]: Same as above, I think eclipse suggested it. I have really no idea what it does, but everything seems to work after the edit.

**Comment by nitnelave** [comment]
Apart from updating the dependencies, I don't see what your changes bring...

**Comment by erik1988** [comment]
well you do have a point. But your current version does not play well with 1.9 and 1.10. So its a need of an update. Also i belive metrics had some issues after 1.9, thats the reason I removed it in my fork. You can probably fix it with no problem, but for me removing it was an easy fix.

Your plugin is great and there is really nothing like it out there, so I think a lot of people would appreciate if you updated it. The changes I made does work, so it is not much work necessary.

Thanks for a great plugin!

**Comment by nitnelave** [comment]
Well, there is a more maintained version here: http://mandrillapp.com/track/click/30152195/minecraft.curseforge.com?p=eyJzIjoiTDhneS1CeERMcWJMaDFnU1BrYnBuaHFESWxJIiwidiI6MSwicCI6IntcInVcIjozMDE1MjE5NSxcInZcIjoxLFwidXJsXCI6XCJodHRwOlxcXC9cXFwvbWluZWNyYWZ0LmN1cnNlZm9yZ2UuY29tXFxcL3Byb2plY3RzXFxcL3Vub2ZmaWNpYWwtZm9yZ2UtY3JlZXBlci1oZWFsXCIsXCJpZFwiOlwiYzkzOTRjNWQxYTgxNDUwYWFkY2NkZTEzNTZlNTVkYjhcIixcInVybF9pZHNcIjpbXCJkMmQ1MmMzMWNkZTUyNDg5NDc2ODAyOWQ0YTlhZTc1ZDY4ZjU1YjRjXCJdfSJ9

**Comment by erik1988** [comment]
Yeah but that's a forge version. I understand that you might not want to spend time updating this much, but im saying its not much work needed to make it work for people on 1.9 and 1.10. Im all good since I use my fork, but I figured this would be helpful for other people if you just released a new version.  Not asking for new features, but I think it would be nice if you uploaded a new version that would work on 1.10 to spigot. Just saying :).

**Comment by nitnelave** [comment]
I'm working on an update, to release with Spigot

**Comment by nitnelave** [comment]
I think I'll include most of your changes (although probably in my own commits). Thanks for your efforts :)

**Comment by erik1988** [comment]
That's great!

**Comment by nitnelave** [comment]
The new version should be what you want.
